resources/newssite/news-site-css/src/css/toggle.css

```diff
@@ -66,9 +69,11 @@
 }
 
 .toggle-container .label input[type="checkbox"]:checked + .switch::before {
-    transform: translateX(20px);
+    transform: translate(20px, -11px);
```

Comment:
Should this be adapted for RTL?

Reply:
this one is weird - it didn't break for me when switching direction to RTL.... does it look weird for you?

Reply:
It doesn't break but the switch still toggles towards the right instead of left in RTL :-)